ddtrace/internal/processor/trace.py

```diff
@@ -101,7 +101,7 @@ class TopLevelSpanProcessor(SpanProcessor):
 
     """
 
-    def on_span_start(self, _):
+    def on_span_start(self, span):
```

Comment:
unnecessary change?

Reply:
Yes, was needed when there were more args but not anymore.